dotnet/samples/KernelSyntaxExamples/Example94_GeminiGetModelResult.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.GoogleVertexAI;
+using RepoUtils;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Examples;
+
+/// <summary>
+/// Represents an example class for Gemini Embedding Generation with volatile memory store.
+/// </summary>
+public sealed class Example94_GeminiGetModelResult : BaseTest
+{
+    [Fact]
+    public async Task GetTokenUsageMetadataAsync()
+    {
+        WriteLine("======== Inline Function Definition + Invocation ========");
+
+        // Create kernel
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddVertexAIGeminiChatCompletion(
+                modelId: TestConfiguration.VertexAI.Gemini.ModelId,
+                bearerKey: TestConfiguration.VertexAI.BearerKey,
+                location: TestConfiguration.VertexAI.Location,
+                projectId: TestConfiguration.VertexAI.ProjectId)
+            .Build();
+
+        // To generate bearer key, you need installed google sdk or use google web console with command:
+        //
+        //   gcloud auth print-access-token
+        //
+        // Above code pass bearer key as string, it is not recommended way in production code,
+        // especially if IChatCompletionService will be long lived, tokens generated by google sdk lives for 1 hour.
+        // You should use bearer key provider, which will be used to generate token on demand:
+        //
+        // Example:
+        //
+        // Kernel kernel = Kernel.CreateBuilder()
+        //     .AddVertexAIGeminiChatCompletion(
+        //         modelId: TestConfiguration.VertexAI.Gemini.ModelId,
+        //         bearerKeyProvider: () =>
+        //         {
+        //             // This is just example, in production we recommend using Google SDK to generate your BearerKey token.
+        //             // This delegate will be called on every request,
+        //             // when providing the token consider using caching strategy and refresh token logic when it is expired or close to expiration.
+        //             return GetBearerKey();
+        //         },
+        //         location: TestConfiguration.VertexAI.Location,
+        //         projectId: TestConfiguration.VertexAI.ProjectId)
+
+        // Create function
+        const string FunctionDefinition = "Hi, give me 5 book suggestions about: {{$input}}";
+        KernelFunction myFunction = kernel.CreateFunctionFromPrompt(FunctionDefinition);
+
+        // Invoke function through kernel
+        FunctionResult result = await kernel.InvokeAsync(myFunction, new() { ["input"] = "travel" });
```

Comment:
Rather than complicating the example with CreateFunctionFromPrompt, this could just use InvokePromptAsync, right?

Reply:
done